B827EB61FA02.json

```diff
@@ -0,0 +1,17 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EB61FA02",
```

Comment:
This ID is invalid, seems to have been manually generated? It should be 16 chars long but is only 12.